src/Config.cpp

```diff
@@ -168,10 +168,34 @@ bool Config::has(const std::string& keyPath)
 
 void Config::initialize()
 {
-    const auto cxxEnv = environment::ConfigProvider::getCxxEnv();
 
+    const auto suppressWarning = std::getenv("SUPPRESS_NO_CONFIG_WARNING");
+
+    // If SUPPRESS_NO_CONFIG_WARNING is defined, do not log the message
+    if (suppressWarning && std::string(suppressWarning) == "1") {
+        // Return early as we don't need to proceed with initialization
+        return;
```

Comment:
How do we know here that config dir is empty? We should use that env variable to not logging warning below if defined.